openfisca_france/parameters/impot_revenu/rvcm/taux_abattement_capitaux_mobiliers.yaml

```
@@ -3,6 +3,6 @@ reference: https://www.ipp.eu/outils/baremes-ipp/
 unit: /1
 values:
   1979-01-01:
-    value: 0.5
+    value: 0.4
   2010-01-01:
     value: 0.4
```

```suggestion
    value: 0.4
```

On peut virer la dernière entrée qui est redondante.